payment_workflow/models/res_partner.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+
+    _inherit = 'res.partner'
+
+    type = fields.Selection([
+        ('contact', 'Contact'),
+        ('invoice', 'Invoice address'),
+        ('supplier_payment', 'Supplier Payment'),
+        ('customer_payment', 'Customer Payment'),
+        ('delivery', 'Shipping address'),
+        ('other', 'Other address')
+    ])
```

Comment:
@yasmineelmrini selection_add=[...]

Reply:
@dufresnedavid There's no problem if the new types are displayed after the old ones ?

Reply:
@yasmineelmrini its not about the order. Its that you don't need to specify types that already exist.

Reply:
@yasmineelmrini no, the order is not important.

Reply:
@dufresnedavid we said that that we wanted the new types right after the invoice type, that's why I specified all those that already exist.
I'll change that now that you're saying that the order is not important :)